lib/assets/javascripts/twine.coffee

```diff
@@ -69,13 +70,29 @@
       element = findOrCreateElementForNode(node)
       element.indexes = indexes
 
-    for type, binding of Twine.bindingTypes when definition = Twine.getAttribute(node, type)
-      element = findOrCreateElementForNode(node)
+    bindingConstructors = null
+    for attribute in node.attributes
+      type = attribute.name
+      type = type.slice(5) if type.slice(0, 5) == 'data-'
+      definition = attribute.value
+
+      constructor = Twine.bindingTypes[type]
```

Comment:
When would `type` ever be `undefined`? You can't have empty attribute key names afaik.